bin/meta-git-remote

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const program = require('commander');
+const debug = require('debug')('meta-git-remote');
+const loop = require('loop-things');
+const path = require('path');
+const util = require('util');
+
+const usages = `\n  usages:\n
+    meta git remote
+    meta git remote -v
+    meta git remote show origin
+    meta git remote rename oldname newname
+    meta git remote rm name`;
+
+if (process.argv[2] === '--help')
+  return console.log(usages);
+
+let first = process.argv[2] === 'blank' ?
+                      process.argv[3] || '':
+                      process.argv[2] || '';
+
+const subCommandParsers = {
+  '-v': function (argv) { return '' },
+  'show': function (argv) {
+    let remote = process.argv[3] === 'blank' ?
+                      process.argv[4] || '':
+                      process.argv[3] || '';
+    return `show ${remote}`;
+  },
+  'rename': function (argv) {
+    
+  },
+  'rm': function (argv) {}
+}
+
+const subCommandParser = subCommandParsers[first];
+
+if ( ! subCommandParser) return console.log(usages);
```

Comment:
thanks. I should be able to find some time to see if I can work on this ticket and then look into the gh plugin. 
One question on this command. How come we have to do the parsing and logic for detection of each of the sub commands? can't we let the actual `git remote` command handle that and if there is an error in user input the default help will be displayed? 

```
Mike@Michaels-MacBook-Pro:chef (master) $ git remote foo
error: Unknown subcommand: foo
usage: git remote [-v | --verbose]
   or: git remote add [-t <branch>] [-m <master>] [-f] [--tags | --no-tags] [--mirror=<fetch|push>] <name> <url>
   or: git remote rename <old> <new>
   or: git remote remove <name>
   or: git remote set-head <name> (-a | --auto | -d | --delete | <branch>)
   or: git remote [-v | --verbose] show [-n] <name>
   or: git remote prune [-n | --dry-run] <name>
   or: git remote [-v | --verbose] update [-p | --prune] [(<group> | <remote>)...]
   or: git remote set-branches [--add] <name> <branch>...
   or: git remote get-url [--push] [--all] <name>
   or: git remote set-url [--push] <name> <newurl> [<oldurl>]
   or: git remote set-url --add <name> <newurl>
   or: git remote set-url --delete <name> <url>

    -v, --verbose         be verbose; must be placed before a subcommand
```